Sources/SwiftGenKit/Parsers/Filter.swift

```diff
@@ -33,6 +33,21 @@ public struct Filter {
   }
 }
 
+extension Parser {
+  /// Create the regex for a filter for the given list of file extensions. The extensions will be matched at the end
+  /// of a filename, and will be matched case-insensitively.
+  ///
+  /// - Parameter extensions: The list of file extensions.
+  /// - Returns: The regex as a `String`.
+  static func filterRegex(forExtensions extensions: [String]) -> String {
+    let extensionsList = extensions.map { NSRegularExpression.escapedPattern(for: $0) }.joined(separator: "|")
+    // - require at least one character that is not a "/" before the "." in order to avoid matching dotfiles
+    // - then a literal "."
+    // - then one of the accepted extensions, in a case-insensitive way (`(?:i…)`), at the end of the string (`$`)
```

Comment:
> in a case-insensitive way (`(?:i…)`), at the end of the string (`$`)

Dammit and it's only now that I've made that suggestion that you already mentioned that last part in the main function's documentation 🤦 

> The extensions will be matched at the end of a filename, and will be matched case-insensitively.

Meh, better repeating ourselves in the doc than not documenting enough I guess, so might as well leave it 🤷‍♂️